contracts/Bond.sol

```diff
@@ -490,53 +492,26 @@ contract Bond is
         @notice the amount of payment tokens required to fully pay the contract
     */
     function amountOwed() public view returns (uint256) {
-        if (totalSupply() <= _upscale(paymentBalance())) {
+        if (totalSupply() <= (paymentBalance())) {
             return 0;
         }
-        uint256 amountUnpaid = totalSupply() - _upscale(paymentBalance());
-        return _downscale(amountUnpaid);
+        uint256 amountUnpaid = totalSupply() - (paymentBalance());
+        return (amountUnpaid);
     }
 
     /**
         @notice gets the amount that was overpaid and can be withdrawn 
         @return overpayment amount that was overpaid 
     */
     function amountOverPaid() public view returns (uint256 overpayment) {
-        if (totalSupply() >= _upscale(paymentBalance())) {
+        if (totalSupply() >= (paymentBalance())) {
             return 0;
         }
-        uint256 amountOverpaid = _upscale(paymentBalance()) - totalSupply();
-        return _downscale(amountOverpaid);
-    }
-
-    /**
-        @dev uses the decimals on the token to return a scale factor for the passed in token
-            tokens that don't implement the `decimals` method are not supported.
-            tokens with more than 18 decimals are not supported
-        @param token the ERC20 token to compute
-        @return scaler above a 1e18 base (1e<decimals> * 1e18)
-    */
-    function _computeScalingFactor(address token)
-        internal
-        view
-        returns (uint256)
-    {
-        uint256 tokenDecimals = IERC20Metadata(token).decimals();
-
-        uint256 decimalsDifference = 18 - tokenDecimals;
-        return 1e18 * 10**decimalsDifference;
-    }
-
-    /**
-        @dev this function takes the amount of paymentTokens and scales to bond
-        tokens. Since the paymentToken may have different decimals than the 
-        bond tokens, scaling to the same base allows calculations between them.
-    */
-    function _upscale(uint256 amount) internal view returns (uint256) {
-        return amount.mulWadUp(_computeScalingFactor(paymentToken));
+        uint256 amountOverpaid = (paymentBalance()) - totalSupply();
+        return (amountOverpaid);
     }
 
-    function _downscale(uint256 amount) internal view returns (uint256) {
-        return amount.divWadDown(_computeScalingFactor(paymentToken));
+    function decimals() public view override returns (uint8) {
```

Comment:
and this be `external`?

Reply:
<img width="499" alt="image" src="https://user-images.githubusercontent.com/15036618/161818605-eff7e861-889d-4449-bac2-af7ec7d602d6.png">